### PR TITLE
Add fetchById method to database

### PR DESCRIPTION
**Pull request by TakLee96**
connects #22 
connects #4

### CODE REVIEW

File: lib/database.js [hunk 1]
```diff
@@ -30,6 +32,11 @@ export default class Database {
     this._cacheStore = new Cache(this.dbID);
   }
 
+  fetchById(id) {
```
Comment (rickmak) [inline lib/database.js]: Let name it getByRecordID. with reference to `getUsersByEmail`.

File: lib/database.js [hunk 2]
```diff
@@ -30,6 +32,13 @@ export default class Database {
...
+    let [recordType, recordId] = id.split('/');
```
Comment (cheungpat) [inline lib/database.js]: use `Record.parseID`

File: lib/database.js (hunk 1 above)
Comment (TakLee96) [inline lib/database.js]: @rickmak How about `getRecordByID`?

**Comment by cheungpat** [comment]
added #4 to the pull request

**Comment by rickmak** [comment]
Merge via 15ffc8a

**Pull request closed without merging**